internal/analytics/client/sync/client.go

```diff
@@ -83,10 +81,6 @@ func New(cfg *config.Instance, auth *authentication.Auth) *Client {
 		a.cfg = cfg
 	}
 
-	if a.cfg.IsSet(constants.ReportAnalayticsConfig) && !a.cfg.GetBool(constants.ReportAnalayticsConfig) {
-		a.sendReports = false
-	}
-
```

Comment:
Please revert, this was intentioanlly done on construction to avoid needless processing and race conditions.

Reply:
Okay.